nuxeo-distribution/nuxeo-launcher/src/main/java/org/nuxeo/launcher/NuxeoLauncher.java

```diff
@@ -1262,6 +1267,48 @@ public void run() {
         }
     }
 
+    protected boolean registerTrial() throws IOException {
+        if (params.length < 2) {
+            printLongHelp();
+            errorValue = EXIT_CODE_ERROR;
+            return false;
+        }
+
+        Console console = System.console();
+        Map<String, String> registration = new HashMap<>();
+        registration.put("email", params[0]);
+        registration.put("company", params[1]);
+
+        String[] pwds = LauncherRegisterHelper.promptPasswords();
+        registration.put("password", pwds[0]);
+        registration.put("password_verif", pwds[1]);
+
+        if (!LauncherRegisterHelper.promptAcceptTerms()) {
+            System.out.println("You must accept the Nuxeo Trial Terms and Condition to register your instance.");
+            errorValue = EXIT_CODE_INVALID;
+            return false;
+        }
+        registration.put("termsAndConditions", "true");
+
+        String projectName = "project1";
+        if (params.length >= 3) {
+            projectName = params[2];
+        }
+        registration.put("connectreg:projectName", projectName);
+
+        try {
+            LauncherRegisterHelper.registerTrial(registration);
+        } catch (RegistrationException e) {
+            System.err.println(e.getMessage() + ":");
```

Comment:
Don't use temp sysout/syserr during development: this class has a few cases of wanted use of sysout instead of log.